smoketest.md

```diff
@@ -0,0 +1,102 @@
+# Smoke Test
+
+For a new system setup, follow the installation steps in [install.md](install.md) 
+
+## Setup
+
+#### Start local Atlas Data Federation
+Set the following environment variables:
+```
+export ADF_TEST_LOCAL_USER=<adf username>
+export ADF_TEST_LOCAL_PWD=<adf password>
+export ADF_TEST_LOCAL_AUTH_DB=<local auth db>
+export ADF_TEST_LOCAL_HOST=<local host address>
+export MDB_TEST_LOCAL_PORT=<MongoDB port>
+```
+Run the following to start a local ADF instance:
+```
+./resources/run_adf.sh start
```

Comment:
I have to modify this script because I installed go with brew, which doesn't put it in the script's expected location. Since go is going to be in the PATH, should we remove that from the script or point it out here?

Reply:
I added a `Golang install location` section to help address this.  The script looks in the expected location for the evergreen hosts.  The `go` executable that is in the PATH is usually not the one we want.